WINDOWS.md

```diff
@@ -52,12 +52,50 @@ python.exe -m venv venv
 . .\venv\Scripts\activate
 ```
 
-**Note**: if you get an error saying 'The term 'python.exe' is not recognized' or 'Python was not found', change 
-`python.exe` in the last line of the above code block to `py.exe` or `python3.exe`. 
-If none of these work, it is likely that Python was not installed on your system. Refer to 
-[this website](https://www.python.org/downloads/windows/) in order to download the latest version. 
+## 1.2 Installation (WSL+Ubuntu)
 
-4. Update `pip` package manager and install `chiadog` dependencies 
+Reminder: `INFO` level log in Chia must be enabled (`chia configure -log-level=INFO`)
+
+1. Open `WSL-Ubuntu`
+
+```
+   git clone https://github.com/martomi/chiadog.git
+   cd chiadog
+```
+
+2. Run the install script.
+
+```
+./install.sh
+```
+
+3. Copy the example config file
+
+```
+cp config-example.yaml config.yaml
+```
+
+4.  Open up `config.yaml` in your editor
+
+    4.1 Set chia log location:
+
+    ```yaml
+    chia_logs:
+      file_log_consumer:
+      enable: true
+      file_path: "/mnt/c/Users/<user-name>/.chia/mainnet/log/debug.log"
+    ```
+
+    4.2 configure it to your preferences for notification services(same as OSX/Ubuntu)
+
+---
+
+**Note**: if you get an error saying 'The term 'python.exe' is not recognized' or 'Python was not found', change
+`python.exe` in the last line of the above code block to `py.exe` or `python3.exe`.
+If none of these work, it is likely that Python was not installed on your system. Refer to
+[this website](https://www.python.org/downloads/windows/) in order to download the latest version.
```

Comment:
I think that note should either be replicated or be higher up closer to the first section for installing natively on windows. Otherwise most people following the instructions with Powershell won't see it.